pkgs/development/python-modules/ftfy/default.nix

```diff
@@ -11,7 +11,7 @@
 buildPythonPackage rec {
   pname = "ftfy";
 
-  version = "4.4.3";
+  version = "5.3.0";
   # ftfy v5 only supports python3. Since at the moment the only
   # packages that use ftfy are spacy and textacy which both support
   # python 2 and 3, they have pinned ftfy to the v4 branch.
```

Comment:
@FRidh What are your thoughts about this?

Reply:
Ah, right. This commit needs to be removed. I've accidentally merged this one before.

Reply:
Done. I've force-pushed to the branch.